model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -29,6 +29,11 @@ import java.io.File
  * The configuration model for all ORT components.
  */
 data class OrtConfiguration(
+    /**
+     * The configuration of the evaluator.
```

Comment:
Should it be ´cli` instead of `evaluator` ?

Reply:
Obsolete, because I removed that commit.